### PR TITLE
Fix tests 0.3

### PR DESCRIPTION
**Pull request by achimgaedke**
* removed the pathlib dependency and test import,
* added min version for python

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by achimgaedke** [comment]
Hey @philip-khor - this should be putting the new `pyprojroot` release through to conda-forge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: "a79900dc52ee097bfd8d917a3d45e2b98494c47a57ba3a71bf609d7a156732e8"
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 109705bb790968704958efcfc5ccce85d8e3dafa054897cc81371fcbbf56cb10
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 1
```
Comment (jamesmyatt) [inline recipe/meta.yaml]: build number should still be 0 since #3 will be closed by this.

**Comment by jamesmyatt** [comment]
@achimgaedke , probably worth you asking to be added as a maintainer too. See https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list

File: recipe/meta.yaml (hunk 1 above)
Comment (achimgaedke) [reply recipe/meta.yaml]: Thank you! Just squshed my commits to one... and here we go.

**Comment by philip-khor** [comment]
@conda-forge-admin please rerender

**Review by philip-khor — APPROVED** [review]

**Comment by philip-khor** [comment]
Thanks for the changes, lgtm. Could you rerender please? I can't seem to push to your fork. 

Feel free to add yourself to the maintainers list @achimgaedke

**Comment by achimgaedke** [comment]
Thanks @philip-khor now i know what's wrong. Will take a look tomorrow.

**Comment by achimgaedke** [comment]
I've replaced this repo with a repo forked from `conda-forge/pyprojroot-feedstock` - instead of the autotick bot repo.

**Comment by achimgaedke** [comment]
The change is minimal, but I hope it is the correct one... (I couldn't find a way of setting the "upstream" remote URL in the GitHub repo, so delete and fork from the correct repo via web frontend...).

image

**Comment by philip-khor** [comment]
No worries, alternately you could do a rerender locally

**Comment by achimgaedke** [comment]
Should be good now, the branch is now forked as recommended...

> Please fork the feedstock directly from conda-forge into your personal GitHub account.

**Review by jamesmyatt — APPROVED** [review]

**Comment by achimgaedke** [comment]
Sorry, running low on time this week and have no rights to merge

image

**Comment by jamesmyatt** [comment]
@achimgaedke , if you open a new issue like #5, then you can request to be added as a maintainer.

**Pull request merged**